### PR TITLE
Added user and drawing models

### PR DESCRIPTION
**Pull request by jerler1**

### CODE REVIEW

**Comment by jerler1** [comment]
Closes #22

File: models/User.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        isAlphanumeric: true,
+        unique: true,
+      },
+    },
+  });
+
+  User.associate = function (models) {
+    // Associating User with Drawings.
+    // When a user is deleted, delete all their drawings.
+    Author.hasMany(models.Drawing, {
```
Comment (brhue) [inline models/User.js:17]: This needs to be `User.hasMany`.

Comment (brhue) [inline models/User.js:17]: This needs to be `User.hasMany`.

File: config/config.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "development": {
     "username": "root",
-    "password": "Mickey19",
+    "password": "kitty1",
     "database": "project2starter",
```
Comment (brhue) [inline config/config.json]: Should probably update database name as well.

**Review by brhue — CHANGES_REQUESTED** [review]
Everything else looks good.

**Comment by jerler1** [comment]
Closes #4

File: config/config.json (hunk 2 above)
Comment (jerler1) [reply config/config.json]: Changed the database name to cobaltCanvasDB in the schema.

File: models/User.js (hunk 1 above)
Comment (jerler1) [reply models/User.js:17]: Changed to User.

**Comment by stopdaydreaming** [comment]
looks good

**Review by stopdaydreaming — APPROVED** [review]
looks good

**Review by brhue — APPROVED** [review]

**Pull request merged**